docs/source/posts/2022/2022-08-17-week-09-shivam.rst

```diff
@@ -0,0 +1,31 @@
+Week 9 - First working skeletal animation prototype
+===================================================
+.. post:: August 16 2022
+   :author: Shivam Anand
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+- This week I had the first working example of skeletal animation ready. I was able to render the ``SimpleSkin`` model. Here's a quick preview:
+
+    .. raw:: html
+
+        <iframe id="player" type="text/html"   width="600" height="390" src="https://user-images.githubusercontent.com/74976752/184981715-a83a8a36-f2bf-4ff1-9366-0c302dcf3f9b.mp4" frameborder="0"></iframe>
+
+- I wrote a custom interpolator (just like the tangent cubic spline interpolator) for interpolating two transformation matrices.
+
+
+What is coming up next week?
+----------------------------
+
+- Generalising the skinning code to work with other models as well.
```

Comment:
Maybe you could explain a bit more about why we need to generalize.

Reply:
By generalizing I meant to support all models. It currently supports only the `simpleskin` model.